lib/pages/dashboard/dashboard_club_teams.dart

```diff
@@ -37,21 +38,23 @@ class _DashboardClubTeamsState extends State<DashboardClubTeams> with SingleTick
 
   @override
   void initState() {
-    _pageController = PageController()
-      ..addListener(() {
-        var nextIndex = _pageController.page.round();
-        setState(() => _currentTeamPage = _pageController.page);
-        if (nextIndex != _currentIndex) {
-          _currentIndex = nextIndex;
-        }
-      });
+    _pageController = PageController()..addListener(() {
```

Comment:
Not related to this PR but it seems your formater and mine are not the same. So each time you/I push these lines change!